tsdfileapi/api.py

```diff
@@ -293,8 +293,8 @@ def enforce_group_logic(self, group_name, group_memberships, tenant, group_confi
             if group_config['enforce_membership']:
                 assert group_name in group_memberships
         except (AssertionError, Exception) as e:
-           logging.error('user not member of group')
-           raise e
+            logging.error(f'Group {group_name} not member of groups {group_memberships}')
```

Comment:
Sorry for the nitpicking here, but the error should be `f'user not member of {group_name}'`. In your previous change I think your intention was to log the user. That would be helpful, but it is not _essential_.

Reply:
Fixed in the last push.

Reply:
+1